algorithms/unit_tests/TestStdAlgorithmsTeamAdjacentFind.cpp

```diff
@@ -55,7 +55,7 @@ struct TestFunctorA {
       case 0: {
         const auto it = KE::adjacent_find(member, KE::cbegin(myRowViewFrom),
                                           KE::cend(myRowViewFrom));
-        Kokkos::single(Kokkos::PerTeam(member), [=]() {
+        Kokkos::single(Kokkos::PerTeam(member), [=, this]() {
```

Comment:
Don't you want
```suggestion
        Kokkos::single(Kokkos::PerTeam(member), [=, *this]() {
```
here and below instead?

Reply:
The `this` pointer points to host data so I find it at the very least confusing to use `this` instead of `*this` here.

Reply:
Windows is not happy with `this` anyway.

Reply:
yes both right, i will fix it